pkgs/dart_services/lib/src/experiments.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// The experiments to use on the main and beta channels.
+const List<String> enabledExperiments = [
```

Comment:
👍

Reply:
fyi @leafpetersen @stereotype441 - please ping us on this repo and or send PRs when the list of supported experiments for dartpad need to change